docs/posts/why-duckdb/index.qmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "Why is DuckDB the default backend for Ibis?"
+author: "Phillip Cloud"
+date: "2024-02-16"
+categories:
+  - blog
+  - community
+---
+
+## Why DuckDB?
+
+Occasionally people ask us why DuckDB is the default backend.
+
+This blog posts aims to address this question.
+
+## A bit of history
+
+For years, Ibis had no default backend. We had somewhat hidden functionality to
+read parquet files with Pandas, but no one working on the project at the time
+was comfortable making Pandas execution the default.
+
+Pandas is great, but we didn't want to inherit all of its [technical
+baggage](https://wesmckinney.com/blog/apache-arrow-pandas-internals/) for
+users, especially new users, of Ibis.
+
+So, we decided not make any backend the official default because our primary
+local backend wouldn't give users a good out-of-the-box experience.
+
+## DuckDB
+
+DuckDB came along as the new kid on the block a few years ago, offering the
+uber-convenience of SQLite but for analytic use cases. It came with excellent
+performance to boot.
+
+It also had a thriving open source community and was built on a solid
+foundation of decades of academic database research, both of which made us all
+optimistic about its future.
+
+It is an **excellent** engine for a use case that is critical for a great
+experience when using Ibis: executing SQL against local data. In particular,
+DuckDB handles larger-than-memory CSV, Apache Parquet and JSON files with
+aplomb.
+
+It was also a big plus that it had entry and exit points for existing ecosystem
+tools like Apache Arrow, and yes, Pandas.
+
+## Fast forward to today
+
+DuckDB is now our the **primary** recommended Ibis backend for working with
+local data.
+
+After we release 9.0, the DuckDB backend will surpass all of our other local
+backends on all fronts--performance and feature set--including `AS OF JOIN`
```

Comment:
```suggestion
backends on all fronts—performance and feature set—including `AS OF JOIN`
```

Reply:
this isn't clear when rendered in monospace, but what I'm suggesting here is to use em dashes ("—")

Reply:
I like a good em dash.  Alternatively, we can also elimate the "all fronts" part and say

"...surpass all of our other local backends in both performance and feature set, including AS OF JOIN support."

Reply:
Went wit @gforsyth's suggestion and removed the emdashes.